packages/shared/src/components/auth/AuthModal.tsx

```diff
@@ -45,6 +46,12 @@ export default function AuthModal({
     closeLogin();
   };
 
+  const onSuccessfulRegistration = (newUser?: LoggedUser | AnonymousUser) => {
+    loginState?.onRegistrationSuccess?.(newUser);
+
+    logSignUp({ experienceLevel: (newUser as LoggedUser)?.experienceLevel });
```

Comment:
I added it globally to all calls but technically inside `logSignUp` method there is a check if pixel tracking is intialized so for example it won't fire from post page. I think thats ok @idoshamun @rebelchris ?